### DISCUSSION TITLE
2022-10-19 Weekly Meeting 78 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#304] [Weekly Meetings]
Weekly Meeting 78 is happening on Wednesday October 19, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Working Group Updates:
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour* 
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- [OMI Website update](https://github.com/omigroup/website)
- [MSF delegates](https://github.com/omigroup/MSF-Delegates) (setting up the repo)
- [M3 avatar interop](https://github.com/M3-org/avatar-interop#calendar)
- [UX Research](https://github.com/omigroup/ux-research) 
- any others?

### Discussions: 
- TBD!

### Future demo themes: (Vote now in the `#omi-vote` Discord channel!)
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo 
- `Generative and AI tooling`
- Any other suggestions?

**Comment by mrmetaverse** [comment]
https://github.com/omigroup/ux-research/issues/12

**Comment by mrmetaverse** [comment]
# Quick exploration with Jin 
:door: [Link](https://hyperfy.io/dawn)

Exploring the street with different Metaverse orgs: 
![image](https://user-images.githubusercontent.com/63426722/196744107-5739d119-e025-4d21-b1c2-f1e1533f8f02.png)

War rooms to display work and ideas: 
![image](https://user-images.githubusercontent.com/63426722/196744286-aa126af5-bb5a-4f01-b2ba-78e2c21bdc2a.png)

"Our brains work spatially" 

![image](https://user-images.githubusercontent.com/63426722/196744465-405597f4-1b12-4b37-91d9-3e9501b326d0.png)

New Event space for Avatar interop
![image](https://user-images.githubusercontent.com/63426722/196744624-cc6926a5-956c-4963-a6e2-f0242862f658.png)

You can portal to other worlds: 
![image](https://user-images.githubusercontent.com/63426722/196744836-b0779aa4-f388-477b-ab5b-de165bf7099c.png)

and always portal back (or hit backspace) 
![image](https://user-images.githubusercontent.com/63426722/196744790-a9882cad-a519-4e06-967a-2bf2af70209b.png)

## Q: Engine?

all in-browser, using react-three-fiber

## Q: Avatars? 

We use VRM! 

## Q: Have a location on the street in mind? 

Yes, currently thinking next to Metafactory. 

## Q: Can we export the street and into other platforms like V-Sekai or ThirdRoom? 

Yes, this is all one GLB! And we are encouraging other collaborations, like implementing the new Open Brush API
![image](https://user-images.githubusercontent.com/63426722/196745777-1daeba31-676b-4200-816c-062e5ce0358d.png)

**Comment by mrmetaverse** [comment]
# UX-Research 
- The group kicked off yesterday! 
  - Read about the [meeting notes here](https://github.com/omigroup/ux-research/discussions/11)
- Next meeting is November 1, 2022 at 16:00UTC 
- How you can help :point_right:  [What are your ideas for problems worth solving?](https://github.com/omigroup/ux-research/issues/12)

Some key considerations: 
- how do we make the metaverse for accessible to people who may be overlooked, or may not be tech-savvy? 
- How do we approach this sustainably? Can we encourage builders/platforms to bring their problem statements worth solving for to collaborate?

**Comment by mrmetaverse** [comment]
# MSF-Delegates 
- some of the MSF domain groups have kicked off and meetings have begun 
  - Wearables was this week 
  - "Real virtual world integration" domain group has their second meeting tomorrow! 
- Next Oversight meeting is coming up (date TBD), and its not sure yet if there will be votes or not.

**Reply by mrmetaverse** [reply]
How is the experience of being a delegate? Can we replicate this in other communities? 

XRGuild ran a hybrid event in both hubs and zoom. Maybe we can imagine more generalized versions of delegates concepts, where available representation from our community attend (and maybe stream from) these other groups.

**Comment by mrmetaverse** [comment]
# Open Metaverse Conference 
- How can we help ensure/influence the conference to have a strong virtual offering? 
- Do we want to create a working group (Events) or create delegates to represent OMI at the conference? 
- We want to ensure more people can get involved and make an impact. (OMI consists of more than chairs, and those who can make it in-person)

**Reply by mrmetaverse** [reply]
Evo Proposed that we get serious about how we approach events, and our presence in virtual spaces (demo days, etc.) and form a Events working group.

**Reply by mrmetaverse** [reply]
Several members have event planning experience, but a call-out is being wary of taking on more than one person can handle.

**Reply by mrmetaverse** [reply]
To prevent this from becoming something a small group controls, how can make sure the broader community has an opportunity to roam in a similar direction? For example, there hasn't been a consensus yet in OMI participation in the event, but now we are talking about forming an event group.

**Reply by mrmetaverse** [reply]
What if we used some sort of bounty system? 
- this was proposed before actually recently, in which Jin gave a demo of dework for this purpose.

**Reply by mrmetaverse** [reply]
We still need to see the language changed, circle back with Jamil: 
![old](https://user-images.githubusercontent.com/63426722/196754409-6858fbbd-4984-43aa-89d2-7ac2dacea94f.png)
# :point_down: 
![NewLang](https://user-images.githubusercontent.com/63426722/196754417-7b48ed49-a4b2-4faa-a6cb-291ec0a3d27f.png)

**Reply by mrmetaverse** [reply]
##  Sindhu called out that we could improve our project management processes. They have volunteered to coordinate any meetings we need to discuss further. (Thank you! :pray:)

### What is the meeting for? 
Defining how we want to participate at some level of consensus. 

### Who needs to be there? 

All members of the community who want to participate in this discussion, and potentially get involved directly with OMI's role in the conference. 

### Outcome: 

A formal proposal for the group to review and discuss regarding the Open Metaverse conference. Use one of the standard templates: 
![image](https://user-images.githubusercontent.com/63426722/196759010-aa5030e5-5a4d-41ea-8dde-cc1a2867958b.png)

### Out of scope: 

Do we form a group of delegates or propose a Events working group? (this is separate)

**Reply by mrmetaverse** [reply]
## Funksoup called out that maybe a delegate structure could work well for this too like we did with MSF. 
Maybe delegates/representatives who can explore potential partnerships/synergy, bring those back to the community as suggestions, and then the community can get involved as a whole. So instead of a working group, there would be designated members. We want to make sure the specific decision-makers will have the time. 

(I think we lack both resources and processes.)

**Reply by mrmetaverse** [reply]
# Mal suggested considering ways to plug our broader community into speaking opportunities based on their interest/expertise. 

OMI as an organization could partake in this or future events, and possibly even have a booth at these events. Other times we may want to send specific experts to represent OMI.

**Reply by mrmetaverse** [reply]
# Tim mentioned: this is one conference of potentially many, so what can we learn from this?

**Reply by mrmetaverse** [reply]
# Funksoup called out: The issue is a beautiful problem. The issue is we are doing a lot, and that's a good problem to have! 
There is so much going on in this community, that some things are stretched a little thin. We can refine our processes as we go.

**Comment by mrmetaverse** [comment]
GREAT POINT
># You don't have to be an elected leader to suggest OMI opportunities and direction

**Comment by mrmetaverse** [comment]
# Evo suggested we check out what Avi implemented for XRGuild for voting.
